.github/workflows/e2e-ci.yaml

```diff
@@ -85,6 +85,7 @@ jobs:
       E2ETestsToRun: ${{ steps.e2e_list_step.outputs.E2ETestsToRun }}
       StopFullCi: ${{ steps.check-if-run.outputs.StopFullCi }}
       TxPower: ${{ steps.set_txpower.outputs.TxPower }}
+      CertRemotePath: "/var/lorastarterkit/certs"
```

Comment:
Why define this as an output and instead of of defining it as a variable on `env` (less indirection)?

Reply:
Adding a variable on env would require more setup, which is actually not strictly required here. I will resolve this and we can consider a refactoring in the future